sys/include/net/gnrc/sixlowpan/frag/sfr/congure.h

```diff
@@ -0,0 +1,420 @@
+/*
+ * Copyright (C) 2021 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup net_gnrc_sixlowpan_frag_sfr
+ * @{
+ *
+ * @file
+ * @brief   Congure definitions for @ref net_gnrc_sixlowpan_frag_sfr
+ *
```

Comment:
Would be great to have some bit of information as to what this does and why/when one would want to enable this.

Reply:
See e4e4fd6ce3